media-sound/jack/files/jack-3.1.1-dontuse-jack-cursesmodule.patch

```diff
@@ -0,0 +1,59 @@
+--- a/jack_t_curses.py	2004-12-12 05:54:36.000000000 +0100
```

Comment:
Since the patch is no longer just about jack_curses, I suppose you'd want to rename it.